README.md

```diff
@@ -6,4 +6,4 @@ To get started simply provide your influxDB address, login, password and databas
 
 Shake your phone to see different changes in timeseries data in your database. 
 
-![Screenshot](https://user-images.githubusercontent.com/3668959/37643733-90480804-2c32-11e8-85c7-193e29baeea4.png)
+![Screenshot](https://raw.githubusercontent.com/CorpGlory/accelerometer2influx/update_screenshot/screenshots/screenshot_upd.png)
```

Comment:
I think it should be moved to the top and centered

Reply:
Do you mean to place the picture above the text?